packages/react-app-polyfill/README.md

```diff
@@ -13,6 +13,8 @@ Each polyfill ensures the following language features are present:
 1. `Symbol` (a built-in object used by `for...of` syntax and friends)
 1. `Array.from` (a built-in static method used by array spread, i.e. `[...arr]`)
 
+*If you need more features, you must include them manually.
```

Comment:
Trailing `*` is missing

Reply:
I actually intended for it to have just the single asterisk, but I'm happy to make the note italic.